bin/sync.sh

```diff
@@ -49,21 +49,13 @@ else
     charts=( $(find $CHART_DIR/ -maxdepth 1 -type d | sed -n "s=$CHART_DIR/\(.\+\)=\1 =p") )
 fi
 
-# install s3 plugin
+# install s3 plugin if not present
 # See https://github.com/hypnoglow/helm-s3/pull/56 for reason to use fork
 s3_plugin_version=$(helm plugin list | grep "^s3 " | awk '{print $2}' || true)
 if [[ $s3_plugin_version != "0.9.0" ]]; then
     echo "not version 0.9.0 from steven-sheehy fork, upgrading or installing plugin..."
     helm plugin remove s3 || true
     helm plugin install https://github.com/steven-sheehy/helm-s3.git --version v0.9.0
-else
-    # double check we have the right version of the s3 plugin
```

Comment:
not sure what the motivation is of removing this. Does it cause any problems? The version info is not specific enough, see also comments in https://github.com/wireapp/wire-server-deploy/pull/112

Reply:
double-checking , to me, is redundant here, but I might lack of a specific use case where this is needed. Furthermore, it uses `$HOME`, which makes some assumption on sth that is not necessarily correct for Helm v3 (`HELM_HOME` couls a) change this, and b) was replaced by those `XDG_[...]` vars).

Reply:
okay, thanks. Well, we can remove this now and in case we get some problems with wrong versions, fix that later.